pages/about.tsx

```diff
@@ -0,0 +1,3 @@
+const About = (props) => <div>About</div>;
+About.layout = "secondary";
+export default About;
```

Comment:
I agree. Maybe there should be there a TypeScript setup. As I'm not myself familiar with it, I'll look into it

Reply:
After looking it up I believe this is the right way. Here we are just assigning a value to layout. The layout is now being set to string type on the LayoutWrapper file. If it's not defined correctly it switches to PrimaryLayout. 
Have a look and if you think think it can be written differently please go ahead and make the change ;)